__tests__/hooks/use-show-warning-secure-account.spec.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react"
-import { useShowWarningSecureAccount } from "@app/screens/settings-screen/show-warning-secure-account"
+import { useShowWarningSecureAccount } from "@app/screens/settings-screen/account/show-warning-secure-account-hook"
 import { renderHook } from "@testing-library/react-hooks"
 import {
   CurrencyListDocument,
```

Comment:
PNG below should be a svg.

Only the screenshot can be a png but everything else (logo, i ona of buttons) should be svg.

Svg are lighter and scale in dimension, which is necessary for mobile

Reply:
Which PNG are you talking about?